### ISSUE TITLE
[1.7.10] Crash with a turbine

### ISSUE THREAD

**Issue by Eufranio** [aidancbrady/Mekanism#3850]
A turbine crashed when i put a conduit in it:

```
java.lang.NullPointerException: Exception in server tick loop
    at mekanism.generators.common.tile.turbine.TileEntityTurbineValve.fill(TileEntityTurbineValve.java:363)
    at crazypants.enderio.conduit.liquid.AdvancedLiquidConduitNetwork.doTick(AdvancedLiquidConduitNetwork.java:150)
    at crazypants.enderio.conduit.liquid.AdvancedLiquidConduitNetwork.access$100(AdvancedLiquidConduitNetwork.java:19)
    at crazypants.enderio.conduit.liquid.AdvancedLiquidConduitNetwork$InnerTickHandler.tickEnd(AdvancedLiquidConduitNetwork.java:337)
    at crazypants.enderio.conduit.ConduitNetworkTickHandler.tickEnd(ConduitNetworkTickHandler.java:47)
    at crazypants.enderio.conduit.ConduitNetworkTickHandler.onServerTick(ConduitNetworkTickHandler.java:35)
    at cpw.mods.fml.common.eventhandler.ASMEventHandler_133_ConduitNetworkTickHandler_onServerTick_ServerTickEvent.invoke(.dynamic)
    at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
    at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at cpw.mods.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:252)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:894)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:693)
    at java.lang.Thread.run(Thread.java:745)
```

Don't know if it is related to EnderIO or Mekanism :/

**Comment by gp009** [comment]
from the looks of the log its likely Ender IO

**Comment by aidancbrady** [comment]
1.7.10 is no longer supported; the fix is included in latest 1.10.2 builds.
